www/index.js

```diff
@@ -742,43 +757,82 @@ async function init() {
     show("#details_custom");
   }
 
-  let versions = await fetch(config.image_url + "/.versions.json", {
-    cache: "no-cache",
-  })
-    .then((obj) => {
-      return obj.json();
+  config.overview_urls = {};
+  config.image_urls = {};
+
+  if (config.mode == "demo") {
```

Comment:
I think this demo mode is a bit over engineered. Can't we just point it to a path which is either a remote URL or a local file? In both cases it'll look for `.versions.json` anyway.

Reply:
Well, it is ugly indeed.

What we could do is make the `.versions.json` mandatory for all setups  (thus removing `versions` and `default_version` from `config.json`). Then we could add an optional image_url in `.versions.json`. That would remove the need for any `mode`.